pcm_win32.cpp

```diff
@@ -24,16 +24,17 @@ class PcmOutputWin32: public PcmOutput {
     Buffer buffers[NBUFS];
     int bufindex;
     int buftail;
+    int sample_rate;
```

Comment:
This should be `const uint32_t` like the rest of them. In fact, since this member is the same for every output class, it should instead be added to the `PcmOutput` class in `pcm.h` (and initialized in the constructor). Then, the `getSampleRate()` member function no longer needs to be `virtual` - the sample rate is just a simple attribute that can be returned immediately by `PcmOutput::getSampleRate()` (and since the attribute is `const`, no derived classes ever need to be concerned about it changing).